modules/test/pom.xml

```diff
@@ -28,6 +24,12 @@
     <dependency>
       <groupId>org.springframework.boot</groupId>
       <artifactId>spring-boot-starter-test</artifactId>
+      <exclusions>
+        <exclusion>
+          <groupId>junit</groupId>
+          <artifactId>junit</artifactId>
```

Comment:
spring boot is depending on JUnit 4 or why do you exclude it here and overwrite the junit version above?

Reply:
spring-boot-starter-test ships with JUnit4 (for compatbility). You can also mix JUnit4 tests and Junit5 tests. However both have their own `@Test` annotation in different packages what caus lots of confusion in auto-completion.
Above I am not adding `junit4` as dependency but entire BOM of junit what contains the versions of all junit4 and junit5 artifacts.

Reply:
ok got it. I thought they already migrated to junit 5

Reply:
@hohwille I have test it though few changes will be required

1. There is a compilation error here as you see the build is failing, 

In class AccessControlSchemaTest
need to imoprt 
import static org.junit.jupiter.api.Assertions.assertEquals;
import static org.junit.jupiter.api.Assertions.assertNotNull;

instead of 
import org.junit.Assert; 

Also required change in method testGroupTypes in same test
according to above imports

2. When we make a project with latest devon4j (with this PR), we still get the same error

What i tried is override the launcher in template devon4j\templates\server\src\main\resources\archetype-resources\core\pom.xml

Currently 1.3.2 is creating the problem

<dependency>
        <groupId>org.junit.platform</groupId>
        <artifactId>junit-platform-launcher</artifactId>
        <version>1.4.2</version>
      </dependency>

With the above changes it worked